ansible/templates/common/cors_configuration.sh.j2

```diff
@@ -0,0 +1,17 @@
+{% if doc %}
+Configure CORS. An ``index`` to specify the ordering of middleware classes is used. It is important to add the ``CorsMiddleware`` as the first class and ``CorsPostCsrfMiddleware`` as the last. You can specify the origin allowed to access or specify all the origin, for example:
+{% else %}
+
+#!/bin/bash
+
+set -e -u -x
+
+export OMERODIR=$OMERODIR
+export PATH={{ virtualenv_path }}/bin:$PATH
+{% endif %}
+
+omero config append omero.web.middleware '{"index": 0.5, "class": "corsheaders.middleware.CorsMiddleware"}'
```

Comment:
Doesn't look like this section is rendering properly:
![Screen Shot 2020-06-03 at 12 06 05](https://user-images.githubusercontent.com/88113/83624244-a62d0d00-a592-11ea-817f-1e62bf6ae2c2.png)

Reply:
not great!!

Reply:
fixed